config/final.yml

```
@@ -1,8 +1,8 @@
 ---
-final_name: s3-broker
+final_name: sqs-broker
 blobstore:
-  provider: s3
+  provider: sqs
   options:
     bucket_name: gds-paas-build-releases-blobs
     region: eu-west-1
-    endpoint: https://s3-eu-west-1.amazonaws.com
+    endpoint: https://sqs.eu-west-1.amazonaws.com
```

```suggestion
    endpoint: https://s3.eu-west-1.amazonaws.com
```

We cannot upload blobs to SQS :D